jboss/container/util/ubi-clean/configure.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
Could you please add:
```
rpm -qa --queryformat "%{NAME}\n" \
    | while read pkg; do microdnf repoquery $pkg 2>/dev/null | grep -q $pkg || echo $pkg; done
```
as a comment here as to how this static list has been determined?